packages/jest-preset/helpers/babel-plugin-import-component.js

```diff
@@ -0,0 +1,51 @@
+module.exports = ({ types: t }) => ({
```

Comment:
I personally would add a brief example of what we're doing here. Just a matter of tastes :).

```suggestion
/*
 * Transforms:
 * import { importComponent } from 'hops';
 *
 * const Foo = importComponent(() => import('./foo'));
 * const Bar = importComponent('./bar');
 *
 * Into:
 * import { importComponent } from 'hops';
 *
 * const Foo = importComponent({component: () => require('./foo')});
 * const Bar = importComponent({component: () => require('./bar')});
 */
module.exports = ({ types: t }) => ({
```

Reply:
Hm… from my experience this team isn't too fond of code comments. Maybe somebody else can decide. 😛 Besides we can still add this later.

Btw: it transpiles to `importComponent({ component: require('./foo') })`.

Reply:
I like the idea, especially in this case because the used code is near impossible to understand without some context.